fs/fs.go

```diff
@@ -128,7 +135,7 @@ func getDockerImagePaths(context Context) []string {
 	// TODO(rjnagal): Detect docker root and graphdriver directories from docker info.
 	dockerRoot := context.DockerRoot
 	dockerImagePaths := []string{}
-	for _, dir := range []string{"devicemapper", "btrfs", "aufs", "overlay"} {
+	for _, dir := range []string{"devicemapper", "btrfs", "aufs", "overlay", "zfs"} {
```

Comment:
Is ZFS a storage driver in docker?
I thought we can use ZFS instead of ext4 as the base fs.